src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Views/Controls/SampleControlInfoAttribute.cs

```diff
@@ -36,6 +45,16 @@ public SampleAttribute(params string[] categories)
 			Categories = categories;
 		}
 
+		/// <summary>
+		/// Marks a class as a sample test control that can be browsed by the SampleChooserControl
+		/// and which can be used by automated tests.
+		/// </summary>
+		/// <param name="categories">An optional list of categories to which this sample is related to</param>
+		public SampleAttribute(params Type[] categories)
+		{
+			Categories = categories.Select(type => type.Name).ToArray();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'categories' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5633443)